docker/local-metadata/docker-compose-postgres.yml

```diff
@@ -0,0 +1,161 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+version: "3.9"
+volumes:
+  ingestion-volume-dag-airflow:
+  ingestion-volume-dags:
+  ingestion-volume-tmp:
+services:
+  postgres:
+    build:
+      context: ../../.
+      dockerfile: docker/local-metadata/Dockerfile_postgres
+    container_name: openmetadata_postgres
+    restart: always
+    depends_on:
+      - elasticsearch
+    environment:
+      POSTGRES_PASSWORD: password
```

Comment:
do we want `POSTGRES_USERNAME` like environment here ? cc - @vivekratnavel

Reply:
Yes, we need username as well in the environment

Reply:
Done